packages/web-app-admin-settings/src/views/Users.vue

```diff
@@ -693,7 +696,7 @@ export default defineComponent({
     updateUserAppRoleAssignments(user, editUser) {
       return this.graphClient.users.createUserAppRoleAssignment(user.id, {
         appRoleId: editUser.appRoleAssignments[0].appRoleId,
-        resourceId: editUser.appRoleAssignments[0].resourceId,
+        resourceId: editUser.appRoleAssignments[0].resourceId || this.applicationId,
```

Comment:
could this always be this.applicationId to avoid confusion ?

Reply:
Should work that way, yes!